Source/Editor/Surface/Archetypes/Material.cs

```diff
@@ -590,7 +591,7 @@ internal enum MaterialTemplateInputsMapping
                 },
                 Elements = new[]
                 {
-                    NodeElementArchetype.Factory.ComboBox(0, 0, 70.0f, 0, FlaxEditor.Tools.Terrain.PaintTerrainGizmoMode.TerrainLayerNames),
+                    NodeElementArchetype.Factory.ComboBox(0, 0, 70.0f, 0, LayersAndTagsSettings.GetCurrentTerrainLayers()),
```

Comment:
Then we can remove `FlaxEditor.Tools.Terrain.PaintTerrainGizmoMode.TerrainLayerNames` if it won't be needed anymore.

Reply:
Done in https://github.com/FlaxEngine/FlaxEngine/pull/3021/commits/c8146a8e99d02f5c1e0bda2023d62189f6c05f07